aep/general/0004/aep.md.j2

```diff
@@ -16,14 +16,40 @@ tools such as Kubernetes or GraphQL interact with APIs from multiple providers.
 APIs **must** define a resource type for each resource in the API, according to
 the following pattern: `{API Name}/{Type Name}`. The type name:
 
+### API Name
+
+An API name is a string that **must** uniquely identify the API. In practice,
```

Comment:
```suggestion
An API name is a string that **must** uniquely identify the API. 

In practice,
```
The first sentence is the most important. Everything else is a rationale for how we think you should determine these based on personal experience.